packages/playwright/index.js

```diff
@@ -13,11 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const path = require('path');
 const {Playwright} = require('playwright-core/lib/server/playwright.js');
 
-module.exports = new Playwright({
-  downloadPath: __dirname,
+const playwright = new Playwright({
   browsers: ['webkit', 'chromium', 'firefox'],
-  respectEnvironmentVariables: true,
 });
+module.exports = playwright;
+
+try {
+  const downloadedBrowsers = require(path.join(__dirname, '.downloaded-browsers.json'));
+  playwright.chromium.setExecutablePath(downloadedBrowsers.crExecutablePath);
+  playwright.firefox.setExecutablePath(downloadedBrowsers.ffExecutablePath);
+  playwright.webkit.setExecutablePath(downloadedBrowsers.wkExecutablePath);
+} catch (e) {
+  throw new Error('ERROR: Playwright did not download browsers');
```

Comment:
What is actually going on when this error is thrown? `.downloaded-browsers.json` does not exist?

Reply:
yes; `.downloaded-browsers.json` does not exist.